openmc/model/surface_composite.py

```diff
@@ -783,6 +784,41 @@ def _make_ccw(self, points):
 
         return points[::-1, :]
 
+    def _constrain_triangulation(self, points):
+        """Generate a constrained triangulation.
```

Comment:
I'd add a sentence as to what "constrained means". 
E.g. "Ensure that all Polygon edges are representable by edges of simplices"

Reply:
done